src/gateway/Gateway.sol

```diff
@@ -2,7 +2,6 @@
 pragma solidity ^0.8.18;
 pragma abicoder v2;
 
-import {TypedMemView} from "memview-sol/TypedMemView.sol";
```

Comment:
Make sure to also remove the submodule in https://github.com/centrifuge/liquidity-pools/blob/main/.gitmodules